applications/irda/irda-app-remote-manager.cpp

```diff
@@ -11,6 +12,8 @@
 #include "irda-app-file-parser.h"
 
 static const std::string default_remote_name = "remote";
+uint32_t const IrdaAppRemoteManager::max_button_name_length = 31;
```

Comment:
`inline constexpr` is more suitable here. See more detailed at the article: https://habr.com/en/company/jugru/blog/506104/